### PR TITLE
added todolist app for week1-react homework

### PR DESCRIPTION
**Pull request by TITHI1244**

### CODE REVIEW

**Review by Ekamba — CHANGES_REQUESTED** [review]
Hi @TITHI1244 , great work so far. But remember when you are rendering a list of items you have to specify a key. You can read more about list key here: https://reactjs.org/docs/lists-and-keys.html

File: react/week1/todo_list/src/Header.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Header = () => {
+    return (
+        <div>
```
Comment (khjcph) [inline react/week1/todo_list/src/Header.js:5]: you actually don't need to put this inside a `div`

File: react/week1/todo_list/src/ListItem.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+function ListItem(props) {
```
Comment (khjcph) [inline react/week1/todo_list/src/ListItem.js:3]: When passing props, you can actually use a very cool javascript feature called destructuring. For example

    const ListItems = ({description, deadline}) => { }

Then you can access the variables without using `props.`

File: react/week1/todo_list/src/App.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    deadline: 'Fri Sep 15 2017'
+  }
+]
+
+function App() {
+  return (
+    <div className="App">
+      <Header />
+      <ul>
+          {todos.map(todo => <ListItem title={todo.title} deadline={todo.deadline} />)}
```
Comment (khjcph) [inline react/week1/todo_list/src/App.js:26]: I don't know if this was covered in class, but lists in react should actually have a unique key. You can read more about those here: https://reactjs.org/docs/lists-and-keys.html#keys. If you would like to you can try to implement keys for your list.

**Review by khjcph — APPROVED** [review]
Great work!! 👍

I left a few comments that you can look at if you want. I don't remember if we covered destructuring in class. If not, you can just ignore that.

**Comment by TITHI1244** [comment]
Thank you for your feedback @Ekamba & @khjcph. I have changed them :)

**Pull request merged**